application/ai/llm_control_service.py

```
@@ -207,6 +211,15 @@ def get_presets(self) -> List[LLMPreset]:
                 description='OpenAI 官方接口（自动兼容底层 Responses API 与 Chat Completions）。导入后请在「模型名」填写所用模型 ID。',
                 tags=['official'],
             ),
+            LLMPreset(
+                key='atlascloud',
+                label='Atlas Cloud',
+                protocol='openai',
+                default_base_url=ATLAS_CLOUD_DEFAULT_BASE_URL,
+                default_model=ATLAS_CLOUD_DEFAULT_MODEL,
+                description='Atlas Cloud OpenAI-compatible 接口，支持统一访问多种文本模型。',
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)